README.md

```diff
@@ -398,10 +398,8 @@ Tests succeeded.
 As with the `build` and `test` commands, you can add custom source paths
 to load, and pass options to the underlying `purs repl` via `--purs-args`.
 
-E.g. the following opens a repl on `localhost:3200`:
-
 ```bash
-$ spago repl --purs-args "--port 3200"
+$ spago repl
```

Comment:
If there are no longer arguments that can be provided to the repl, should this section be reworded or removed altogether?

Reply:
I'd remove it. Even if it technically would still work on `purs-0.14`, it's not a well-known feature.

Reply:
I rephrased it since the `repl` command is still valid.